Binary_Tree_TopView/Binary_Tree_TopView.cpp

```diff
@@ -0,0 +1,81 @@
+#include <iostream>
+#include <queue>
+#include <map>
+using namespace std;
+
+// Structure of binary tree
+struct Node
+{
+    Node *left;
+    Node *right;
+    int hd;
```

Comment:
Can you change it to a more meaningful variable name?

Reply:
Sure!